Reading List/Reading List/ReadingListTableViewController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  ReadingListTableTableViewController.swift
+//  Reading List
+//
+//  Created by James Sedlacek on 6/5/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class ReadingListTableViewController: UITableViewController {
+    
+    let bookController: BookController = BookController()
+
+    override func viewDidLoad() {
+        bookController.loadFromPersistentStore()
+        super.viewDidLoad()
+        tableView.delegate = self
+        tableView.dataSource = self
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        bookController.loadFromPersistentStore()
+        tableView.reloadData()
```

Comment:
Great job figuring out where to put a reload to refresh your data when returning to this screen. viewWillAppear is also an option that is called slightly earlier. You probably don't need to load from the persistent store in this method though, as you have already loaded in viewDidLoad